src/test/java/com/pwn9/filter/minecraft/command/PwnFilterCommandExecutorTest.java

```diff
@@ -0,0 +1,43 @@
+package com.pwn9.filter.minecraft.command;
+
+import com.pwn9.filter.MockMinecraftAPI;
+import com.pwn9.filter.MockPlayer;
+import com.pwn9.filter.MockPlugin;
+import com.pwn9.filter.bukkit.PwnFilterPlugin;
+import com.pwn9.filter.engine.FilterService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Queue;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created for use for the Add5tar MC Minecraft server
+ * Created by benjamincharlton on 20/08/2020.
+ */
+public class PwnFilterCommandExecutorTest {
+    private final PwnFilterPlugin testPlugin = new MockPlugin();
+    private final MockPlayer testPlayer1 = new MockPlayer();
+    private final MockPlayer testPlayer2 = new MockPlayer();
+
+
+    @Before
+    public void Setup(){
```

Comment:
Codacy found an issue: [The instance method name 'Setup' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/AddstarMC/PwnFilter/pullRequest?prid=6040651)